src/components/Nav/index.tsx

```diff
@@ -25,9 +26,9 @@ const Nav: FC<IProps> = ({ path }) => {
     t,
     toggleColorMode,
     linkSections,
-    searchRef,
     mobileNavProps,
   } = useNav({ path })
+  const searchModalDisclosure = useDisclosure()
```

Comment:
Moved the search modal logic to the parent to avoid using refs which was causing me a headache.

Reply:
Good call here 😅